dashboard/config/levels/custom/dance/dance_ai_generate_effect.level

```diff
@@ -53,8 +53,8 @@
     "name_suffix": "_draft3",
     "authored_hints": "[{\"hint_class\":null,\"hint_markdown\":\"💡 Connect the <xml><block xmlns=\\\"https://developers.google.com/blockly/xml\\\" type=\\\"Dancelab_ai\\\" id=\\\"}Kh?+Ed9eixX[?tgg{+h\\\" disabled=\\\"true\\\">\\n  <field name=\\\"VALUE\\\"></field>\\n</block></xml> into <code style=\\\"color:white; background-color:#FFBD46\\\">setup</code>. Then click on it to get started.\",\"hint_id\":\"Hint2\",\"hint_type\":\"general\",\"hint_path\":\"\",\"hint_video\":\"\",\"tts_url\":\"https://tts.code.org/sharon22k/180/100/e6636a859977f9f6d93d7d7d25d3e729/draft_ai_dance_party_drag_ai_cosmic_draft3.mp3\"},{\"hint_class\":null,\"hint_markdown\":\"💡 If you want to regenerate the effect or change the emojis you can click on the `A.I. generate effect` block again. \",\"hint_id\":\"Hint3\",\"hint_type\":\"general\",\"hint_path\":\"\",\"hint_video\":\"\",\"tts_url\":\"https://tts.code.org/sharon22k/180/100/13f2bc429620981c1ae27fc2c7147d2a/draft_ai_dance_party_drag_ai_cosmic_draft3.mp3\"}]",
     "default_song": "summer_calvinharris",
-    "long_instructions": "**A.I. is here to find the best effect for your dance party!**\r\n\r\n<img src=\"https://images.code.org/4a3bd15e97f6dd79f9a57fc07ad79017-image-1697662602704.png\" style=\" margin-right: 40px;  height: 100px; float: left\">\r\n\r\n- Connect the <code style=\"color:white; background-color:#566064\">AI generate effect</code> block to setup.\r\n\r\n- Click on the block to get started. \r\n",
-    "short_instructions": "**A.I. is here to find the best effect for your dance party!**\r\nConnect the `AI generate effect` block to setup then click on the block to get started.",
+    "long_instructions": "**A.I. can help you create fun visual effects for your dance party!**\r\n\r\n<img src=\"https://images.code.org/4a3bd15e97f6dd79f9a57fc07ad79017-image-1697662602704.png\" style=\" margin-right: 40px;  height: 100px; float: left\">\r\n\r\n- Connect the <code style=\"color:white; background-color:#566064\">AI generate effect</code> block to setup.\r\n\r\n- Click on the block to get started. \r\n",
+    "short_instructions": "**A.I. can help you create fun visual effects for your dance party!**\r\nConnect the `AI generate effect` block to setup. Then click on the block to get started.",
```

Comment:
no preference on my end just confirming the inconsistency in A.I. v. AI is expected

Reply:
Thanks! I'm going to check with the team about this.

Reply:
Thanks so much Erin! We do want consistency so I updated in three levels.